docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -62,17 +80,20 @@ In general, there are three types of privileges you'll work with:
 ***********************************  ***********************************
 ////
 
+[float]
 [[apm-privileges-to-publish-events]]
-=== Grant privileges and roles needed for writing events
+=== Create a _writer_ role
 
-++++
-<titleabbrev>Create a _writer_ user</titleabbrev>
-++++
 
-APM users that publish events to {es} need privileges to write to APM data streams.
+APM users that publish events to {es} _must_ privileges to write to APM data streams.
 
-[float]
-==== General writer role
+// Not sure if I captured this accurately...
+[NOTE]
+====
+This is not needed if APM Server doesn't write to {es} directly.
+Instead, another configured {es} role will need to be passed to
+<<apm-agent-config-elasticsearch,`apm-server.agent.config.elasticsearch`>>.
+====
```

Comment:
This doesn't sound right to me, but I may be missing some use case. I have 2 points:
1. APM Server would always write to ES directly :thinking: What use cases would not see APM Server using the Elasticsearch output but not writing directly to Elasticsearch?
2. Documentation for `output.elasticsearch`, which `apm-server.agent.config.elasticsearch` link to, does not mention roles at all, so I think this ends up being too confusing.

Can we clarify (1)? I think (2) will be more easy to explain after clarifying that.

Reply:
> APM Server would always write to ES directly

APM server can be configured to write to logstash, kafka etc, all the outputs supported by libbeat.

Reply:
I attempted to add more context related to (1) in 571a58ca34b28e8339ba532ebc45fbd57d2ceb97, but it's not clear to me what needs to be done related to (2). @endorama @carsonip can you be more specific about what should be changed about the note as it exists in 571a58ca34b28e8339ba532ebc45fbd57d2ceb97?